includes/class-system-info.php

```
@@ -0,0 +1,297 @@
+		 * }
+		 */
+		public static function get_database(): array {
+			global $wpdb;
+
+			return array(
+				'server_version'  => $wpdb->db_version() ? $wpdb->db_version() : null,
+				'extension'       => isset( $wpdb->dbh ) ? get_class( $wpdb->dbh ) : null,
+				'client_version'  => $wpdb->dbh->client_info ?? null,
+				'max_connections' => self::get_mysql_var( 'max_connections' ),
+			);
+		}
```

PHPMD (2.15.0)

[error] The method get_database is not named in camelCase. (undefined)

(CamelCaseMethodName)